cli/conf.py

```diff
@@ -53,7 +75,16 @@ def parse_args(cls, module_name, config, args=None):
         :param module_name: the module name: installer|provisioner|tester
         """
         cmd = clg.CommandLine(cls._get_specs(module_name, config))
-        return cmd.parse(args)
+        res_args = cmd.parse(args)
+
+        # override the command specific arguments from file
```

Comment:
We don't want to override CLI from file.

the search order should be:
1. look for item in CLI.
2. If not found in CLI, look in file (if provided)